image-specs/el7.3-base

```diff
@@ -0,0 +1,16 @@
+#base=libguestfs:centos-7.3
+#name=el7.3-base
+#osinfo=centos7.3
+#distro=el7
+#version=v1
+#arch=x86_64
+#expand=/dev/sda2
+#lvexpand=/dev/cl/root
+#root-partition=/dev/cl/root
+root-password password:123456
+update
+install qemu-guest-agent
+
+# fix for bz#1302678
```

Comment:
I vote for 1.
We certainly don't want to delay the relabeling to Lago, as I hope to get rid of it there!